src/prerender.ts

```diff
@@ -61,12 +62,12 @@ const prerenderRoute = (shape, stationIds) => {
     };
 
     entries.forEach((entry) => {
-        if (entry.type === 'stationRange') {
+        if (typeof entry !== 'function' && entry.type === 'stationRange') {
```

Comment:
Just curious why this type check?

Reply:
So there are two things passed in, either an object or a function. And the functions don't have a `type`, so they were making TS angry. The resulting check here is the same, but we now fail quicker if it's a function and Typescript is happy